cms/static/cms/js/modules/cms.page_type_select.js

```diff
@@ -0,0 +1,26 @@
+//##############################################################################
```

Comment:
This JS file is part of a specific "widget" for a specific field for a specific cms_wizard. It doesn't make sense to combine it with anything else. This should *not* be included on all cms_wizard forms.